core-web/package.json

```diff
@@ -258,7 +264,14 @@
         "tslint-angular": "^3.0.3",
         "typedoc": "^0.25.4",
         "typescript": "5.3.3",
+        "verdaccio": "^5.0.4",
+        "vite": "~5.0.0",
+        "vite-plugin-dts": "~2.3.0",
+        "vitest": "^1.0.4",
```

Comment:
Not sure why we're going with Vite if all the other sdk libs are being builded with `tsc`, I vote we keep it consistent.

Reply:
I tried tsc, esbuild, swc, rollout and the latest, vite. With Vite the only configuration I had to do was define how I wanted the library (iife) without having to install other things or make complex configurations (which didn't work for me).

```
  lib: {
            // Could also be a dictionary or array of multiple entry points.
            entry: 'src/lib/standalone.ts',
            name: 'Experiments',
            fileName: 'dot-experiments',
            // Change this to the formats you want to support.
            // Don't forget to update your package.json as well.
            formats: ['iife']
        },
```
Other format accepted ` 'umd', 'cjs', "es"`.

But, I can try to use tsc plain anyway.

Reply:
remove `vite`, to use `rollout` with `swc`.